api/pom.xml

```diff
@@ -41,7 +41,7 @@
         <dependency>
             <groupId>com.comphenix.protocol</groupId>
             <artifactId>ProtocolLib</artifactId>
-            <version>4.8.0</version>
+            <version>5.0.0-SNAPSHOT</version>
```

Comment:
Version `5.0.0-SNAPSHOT` doesn't appear to exist anymore as an official ProtocolLib version. `5.0.0` seams to work fine.